actor-typed/src/main/resources/reference.conf

```diff
@@ -26,7 +26,7 @@ pekko.actor.typed {
   # buffer. If the capacity is exceed then additional incoming messages are dropped.
   restart-stash-capacity = 1000
 
-  # Typed mailbox defaults to the single consumer mailbox as balancing dispatcher is not supported
+  # Typed mailbox defaults to the single consumber mailbox as balancing dispatcher is not supported
```

Comment:
`consumer` was correct

Reply:
Thanks, that's previous commit and a rebase operation messed up these file change.